CONTRIBUTING.md

```diff
@@ -118,43 +115,31 @@ Follow the procedure below to contribute to the Strapi documentation.
 
-#### Handling
+✋ Sign the CLA (Contributor License Agreement), directly via GitHub (see [our dedicated blog post](https://strapi.io/blog/switching-from-dco-to-cla) for more information).
 
-When a new pull request is submitted in the Strapi `documentation` repository, it is handled in the following week, meaning that the Strapi team:
+### Review and management of pull requests
 
-- acknowledges the new pull request, leaving a 👀 reaction on the description
-- tags the pull request, to indicate the type of documentation that is being updated, and optionally the level of priority for the contribution
-- assigns the pull request to a member of the Strapi Documentation team, who will be in charge of the reviewing and merging
+The pull request review process and timeline are based on the availability of Strapi's Documentation team to handle community contributions. The workflow is:
 
-#### Review
+1. The pull request is assigned to a member of the Documentation team.
+2. At least 1 member of the Documentation team will review the pull request for:
 
-Once all of the handling steps are done by the Strapi team, you can expect your pull request to be reviewed in the next week or so.
+- accuracy,
+- quality,
+- alignment with the documentation scope and roadmap.
 
-Here is what the Strapi Documentation team will review in your pull request:
-
-- The accuracy and coherence of the contribution, by testing any new piece of code or action for example,
-- The quality of the technical writing, to make sure it is proper technical documentation that is easy to understand (see [12 Rules of Technical Writing](https://www.notion.so/12-Rules-of-Technical-Writing-c75e080e6b19432287b3dd61c2c9fa04))
-- The fit of the contribution among the rest of the Strapi documentation (see [Strapi Documentation Style Guide](https://www.notion.so/Strapi-Documentation-Style-Guide-b9c21387732741ed8b0a3f9701612577))
-
-The pull request will be validated once the contribution ticks all the boxes. At least the validation of the Strapi Documentation team member in charge of the pull request is necessary for it to be merged.
-
-#### Deployment
-
-Once your pull request is validated and merged, it will be deployed with the next batch of merged pull requests. The Strapi Documentation team only deploys the documentation on Wednesdays, but not every week.
+3. Reviewers will either approve, ask for changes, or reject the pull request.
+4. Accepted pull requests will be merged and deployed as part of the regular documentation deployments on Wednesdays, but not every week.
```

Comment:
I'd say we now have a stable schedule: patch release every 2 weeks in sync with product releases. So we could probably update the sentence:
```suggestion
4. Accepted pull requests will be merged and deployed as part of the regular documentation deployments on Wednesdays every 2 weeks.
```

Reply:
Thanks Pierre, that is a great suggestion and it reads better too.